docs/hardware.rst

```diff
@@ -513,17 +513,22 @@ instance we recommend using 7th- or 8th-generation NUCs instead.
 2014 Mac Minis
 ~~~~~~~~~~~~~~
 
+.. important::
+
+   SecureDrop 2.2.0 will be the last regular release of SecureDrop
```

Comment:
Question about versioning: Would we be dropping Mac Mini support in 2.2.1 then? Or would we do that in 2.3.0?

Reply:
I think we would default to attempting to continue support Mac Minis for any 2.2.x, while dropping official support for 2.3.0 (it may not be breaking just yet, but we would no longer test against that hardware). We could make that more explicit and say "The SecureDrop 2.2.x series" or similar.

Reply:
Your suggested wording sounds good to me!